modules/lvgl/Kconfig

```diff
@@ -128,15 +130,45 @@ config LV_Z_AREA_Y_ALIGNMENT_WIDTH
 	  the current frame dimensions to meet display and/or LCD host
 	  controller requirements. The value must be power of 2.
 
-config LV_USE_GPU_STM32_DMA2D
+config LV_Z_AUTO_INIT
+	bool "Initialize LVGL before application startup"
+	default y
+	help
+	  Configure LVGL callbacks and display initialization before the application starts.
+	  This can be useful to disable if you need to change a display's pixel format
+	  prior to initialization. If using static allocation, ensure that LV_Z_BITS_PER_PIXEL
+	  is set correctly.
+
+config LV_USE_DRAW_DMA2D
 	bool
 
-config LV_GPU_DMA2D_CMSIS_INCLUDE
+config LV_DRAW_DMA2D_HAL_INCLUDE
 	string
 	help
 	  Must be defined to include path of CMSIS header of target processor
 	  e.g. "stm32f769xx.h" or "stm32f429xx.h"
 
+config LV_Z_USE_OSAL
+	bool "Use OSAL enabling parallel rendering"
+	depends on DYNAMIC_THREAD
+	help
+	  Use the Zephyr LVGL OSAL to enable parallel rendering
+	  pipelines.
+
+config LV_USE_PXP
+	bool
```

Comment:
Should this be promptless? Doesn't it make sense to allow disabling the PXP?

Reply:
Yeah the reason for this symbol is that LVGL v9 https://github.com/lvgl/lvgl/blob/master/Kconfig#L384 names it `LV_USE_PXP`, former versions used `LV_USE_GPU_NXP_PXP` and at some point made its way into hal_nxp https://github.com/zephyrproject-rtos/hal_nxp/blob/6ac83c6ba14830871e19db7bd4c30af8b577fce2/mcux/hal_nxp.cmake#L218.

So for backwards compatibility to the hal layer this was added as a forward declared prompt so that the old symbol can be enabled:
```
config LV_USE_GPU_NXP_PXP
	bool
	default y if LV_USE_PXP
```